docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md

```diff
@@ -26,6 +27,512 @@
 | [v1.7.1](Data_Dictionary_Release_Notes.md#release-with-api-v171) | March 16, 2017 |
 | [v1.3.1](Data_Dictionary_Release_Notes.md#release-with-api-v131) | September 7, 2016 |
 
+## v3.1.0
+
+* __GDC Product__: GDC Data Dictionary
+* __Release Date__: May XX, 2024
+
+### New Features and Changes
+
+* New Entity: `germline_mutation_index`
+* New Entity: `submitted_expression_array`
+* Altered `aggregated_somatic_mutation` Entity
+	* New property: `platform`
+* Altered `aligned_reads` Entity
+	* Added `minimum` and/or `maximum` values to properties <!-- summarized this for readability --> 
```

Comment:
Make sure this note doesn't appear in the text.  For some reason the syntax highlighting didn't catch this as a comment (as it did the one after).  Highlighting isn't always dependable, but it's worth a check.

Reply:
Yeah I saw this too, in my local version of the markdown file it is greyed out indicating comment. I will double check this when we test QA deployment of docs